app/assets/stylesheets/layout/_work_packages_full_view.sass

```diff
@@ -153,6 +166,8 @@ body.controller-work_packages.full-create
     @at-root
       .detailsViewMenuItem
         display: block
+  .work-packages-full-view--resizer
```

Comment:
That is the wrong breakpoint. It should only vanish from 1248px down. Please move this to the file `work_package_mobile.sass`

Reply:
Done!